kbclient/account/account_client.go

```diff
@@ -6,37 +6,15 @@ package account
 // Editing this file might prove futile when you re-run the swagger generate command
 
 import (
-	"context"
 	"fmt"
 
 	"github.com/go-openapi/runtime"
-	"github.com/killbill/kbcli/v2/kbcommon"
-
-	strfmt "github.com/go-openapi/strfmt"
+	"github.com/go-openapi/strfmt"
 )
 
 // New creates a new account API client.
-func New(transport runtime.ClientTransport,
-	formats strfmt.Registry,
-	authInfo runtime.ClientAuthInfoWriter,
-	defaults KillbillDefaults) *Client {
-
-	return &Client{transport: transport, formats: formats, authInfo: authInfo, defaults: defaults}
-}
-
-// killbill default values. When a call is made to an operation, these values are used
-// if params doesn't specify them.
-type KillbillDefaults interface {
-	// Default CreatedBy. If not set explicitly in params, this will be used.
-	XKillbillCreatedBy() *string
```

Comment:
I think the custom templates weren't taken into account, as this section of the code is missing now: https://github.com/killbill/go-swagger/blob/killbill/generator/templates/client/client.gotmpl#L41

Maybe the step 1 from the README failed?

```
go1.13 generate ./generator
```

Until we have CI for this, the easiest way to verify the new build would be to run the `kbcmd` tutorial from https://github.com/killbill/kbcli/blob/master/kbcmd/README.md

Reply:
FYI, filed:
* https://github.com/killbill/kbcli/issues/29
* https://github.com/killbill/kbcli/issues/30
* https://github.com/killbill/kbcli/issues/31

Reply:
Hi @pierre,

I have regenerated the client on linux machine. Please review if it looks ok now.

Updated README.md too.

Reply:
Hi @pierre could you please review the latest changes. 

adding @reshmabidikar